src/__tests__/setup.ts

```
@@ -0,0 +1,425 @@
+      }
+    })
+  )
+}
+
+// Mock Prompt Service  
+export interface MockPromptService {
+  readonly select: <A>(options: { message: string, choices: Array<{ title: string, value: A }> }) => Effect.Effect<A>
+  readonly setResponse: <A>(response: A) => Effect.Effect<void>
+}
+
+export class MockPromptService extends Context.Tag('MockPromptService')<
```

Biome (2.1.2)

[error] This class is unsafely merged with an interface.

The interface is declared here.

The TypeScript compiler doesn't check whether properties defined in the interface are initialized in the class.

(lint/suspicious/noUnsafeDeclarationMerging)